Library/Formula/unbound.rb

```diff
@@ -19,13 +19,10 @@ def install
                           "--sysconfdir=#{etc}",
                           "--with-libevent=#{Formula["libevent"].opt_prefix}",
                           "--with-ssl=#{Formula["openssl"].opt_prefix}"
+    inreplace "doc/example.conf", 'username: "unbound"', "username: \"#{ENV["USER"]}\""
```

Comment:
I think the `inreplace` needs to remain conditional, on the principle that if the user has changed the config file, we shouldn't go changing it further? A `brew install` run should only change that file if it was placed there by that same run. Same reason we don't overwrite existing files in `etc`.

Anyway, /me smacks forehead for not realizing why it won't work.